x-pack/legacy/plugins/siem/public/components/bytes/index.tsx

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import * as React from 'react';
+import { pure } from 'recompose';
+
+import { DefaultDraggable } from '../draggables';
+import { PreferenceFormattedBytes } from '../formatted_bytes';
+
+export const BYTES_FORMAT = 'bytes';
+
+/**
+ * Renders draggable text containing the value of a field representing a
+ * duration of time, (e.g. `event.duration`)
+ */
+export const Bytes = pure<{
```

Comment:
I'd use a `React.memo` here. With the latest update to react-redux from a few days ago, we are able to even use `React.memo` for redux components.